### ISSUE TITLE
Large renders crash Fractorium.

### ISSUE THREAD

**Issue by perrelet** [mfeemster/fractorium#89]
Fractorium crashes without throwing an error when I try and render larger flames (approx >6000 px²). Is anyone else having this issue? Could it perhaps be a memory overload problem?

Would be great to hear if anyone else is seeing the same behavior?

**Comment by mfeemster** [comment]
Please provide the amount of RAM and VRAM available on your system.

Where is the crash happening? Are you trying to render an image of that size in the main window, or in the final render dialog? Remember that the main window is using an OpenGL texture to display the image, so your card might not support textures that large. If that is the case, set it to a reasonable size for editing, then use the scale feature in the final render dialog to get it to the size you want.

**Comment by perrelet** [comment]
Specs:
RAM: 16GB DDR3
VRAM: 2GB DDR3

The crash is happening from the final render dialog - I am using the scale feature to increase the resolution. After clicking start the app freezes and closes.

The strips feature - great idea - can really go massive then! ;)

**Comment by mfeemster** [comment]
Yeah, you obviously can't fit massive images into only 2GB of VRAM. As you are adjusting the sizes, do you see warning text at the bottom that tells you that you will exceed memory?

Let me know if strips worked, and what number of them you used?

I will be documenting this on the upcoming website.
